scrapy/downloadermiddlewares/httpproxy.py

```diff
@@ -17,15 +17,18 @@ class HttpProxyMiddleware(object):
     def __init__(self, auth_encoding='latin-1'):
         self.auth_encoding = auth_encoding
         self.proxies = {}
-        for type, url in getproxies().items():
-            self.proxies[type] = self._get_proxy(url, type)
+        for type_, url in getproxies().items():
+            self.proxies[type_] = self._get_proxy(url, type_)
 
     @classmethod
     def from_crawler(cls, crawler):
         if not crawler.settings.getbool('HTTPPROXY_ENABLED'):
             raise NotConfigured
+
         auth_encoding = crawler.settings.get('HTTPPROXY_AUTH_ENCODING')
-        return cls(auth_encoding)
+        obj = cls(auth_encoding)
+
+        return obj
```

Comment:
I'm not sure changes to this method add to clarity. pep8 says: "Use blank lines in functions, **sparingly**, to indicate logical sections." (emphasis is mine).

Reply:
Agreed, I will revert this change.